### DISCUSSION TITLE
看样子是看流量大小，直接开始封端口了

### DISCUSSION THREAD

**Discussion by shenlijun** [XTLS/Xray-core#623] [General]
RT

**Comment by GWONGJYULEI** [comment]
+1
用的是VLESS+XTLS+WEB

**Comment by asiaon123** [comment]
大家都有解么

**Comment by galaxyskyknight** [comment]
无解，只能不看奈飞，纯上上网

**Comment by qiangliu8183** [comment]
这几天要疯，一直稳定使用5年的vps都给我封了，只封443
希望如你所说是看流量大小封

**Comment by shenlijun** [comment]
也可能是针对域名了
昨天刚换的IP，几乎什么都没干，443又被封了

**Reply by galaxyskyknight** [reply]
主动探测吧，除了流量大小特征本身外，我总感觉xtls协议本身可能还有比较显著的特征或哪里不太完善了，我现在换回tls+figerprint random，目前还活着。

**Reply by shenlijun** [reply]
我去试一试@@

**Reply by shenlijun** [reply]
流量一大，立马暴毙，咔咔

**Reply by qiangliu8183** [reply]
> 流量一大，立马暴毙，咔咔

又被办了1个，应该和流量没关系，看4K视频时没事，逛github时突然被办的，现在换回v2ray wss了

**Reply by shenlijun** [reply]
> > 流量一大，立马暴毙，咔咔
> 
> 又被办了1个，应该和流量没关系，看4K视频时没事，逛github时突然被办的，现在换回v2ray wss了

这个肯定有个延迟嘛

**Comment by AstralHope** [comment]
差不多就是这两天开始的，共享的几台机器也都是封了特定的端口，私人专用就没事，只用了tcp+xtls，现在还有哪种方法更靠谱呀？

**Reply by qiangliu8183** [reply]
> 差不多就是这两天开始的，共享的几台机器也都是封了特定的端口，私人专用就没事，只用了tcp+xtls，现在还有哪种方法更靠谱呀？

和共享应该没关系，我这里共享的v2ray就没问题，严重怀疑是xtls的问题

**Reply by galaxyskyknight** [reply]
我也觉得xtls应该是有某种还未知的可探测特征（非流量大小特征）已被墙掌握且能被墙侦测，纯从各大坛子的样本统计反馈猜测的。

**Reply by AstralHope** [reply]
> > 差不多就是这两天开始的，共享的几台机器也都是封了特定的端口，私人专用就没事，只用了tcp+xtls，现在还有哪种方法更靠谱呀？
> 
> 和共享应该没关系，我这里共享的v2ray就没问题，严重怀疑是xtls的问题

说共享的意思主要是指用的人多，之前vmess ws+tls也有一台机器没封过IP，于是才升的vless（仅此一例，其他完全没碰到过），没想到还没来得及全面转型就开始出问题了

**Reply by shenlijun** [reply]
> 我也觉得xtls应该是有某种还未知的可探测特征（非流量大小特征）已被墙掌握且能被墙侦测，纯从各大坛子的样本统计反馈猜测的。

确实有可能
@rprx

**Reply by yrong** [reply]
是的xtls会封端口，可以套个cf的壳子走websocket

**Reply by AstralHope** [reply]
> 是的xtls会封端口，可以套个cf的壳子走websocket

套cf太慢了，封端口就不用了呗，换个端口，不开xtls了，直接tcp+tls或者ws+tls

**Reply by AstralHope** [reply]
> 是的xtls会封端口，可以套个cf的壳子走websocket

现在主要是需要有大佬说明下被封的可能原因和补救措施，不开xtls的话vless的优越性降低了呀

**Reply by qiangliu8183** [reply]
> > 是的xtls会封端口，可以套个cf的壳子走websocket
> 
> 套cf太慢了，封端口就不用了呗，换个端口，不开xtls了，直接tcp+tls或者ws+tls

不开xtls还用啥xray，直接回v2ray不好么

**Reply by AstralHope** [reply]
> > > 是的xtls会封端口，可以套个cf的壳子走websocket
> > 
> > 
> > 套cf太慢了，封端口就不用了呗，换个端口，不开xtls了，直接tcp+tls或者ws+tls
> 
> 不开xtls还用啥xray，直接回v2ray不好么

是啊，所以就很蠢，开了会封，不开跟v2ray一样，等大佬解答，看看下个版本能不能有解了
